src/gas/models/Club/RegistModel.ts

```diff
@@ -20,13 +21,16 @@ export default class RegistModel {
         カラム定義: /docs/columns/clubs.json, /docs/columns/clubs.png
         */
         const id = Utilities.getUuid();
+        const slackChannelUrl = `${slackWorkspaceUrl}/archives/${club.channelId}`;
+        const slackChannel = `= HYPERLINK("${slackChannelUrl}", "${club.channelId}")`;
```

Comment:
```suggestion
        const slackChannel = `=HYPERLINK("${slackChannelUrl}", "${club.channelId}")`;
```
スペース入ってたけど機能してる？大丈夫？

Reply:
修正します！癖で入れてしまってました！

一応空白はあってもなくても機能はするみたいですが、修正します。